sdk/Trace/SpanOptions.php

```diff
@@ -92,7 +92,7 @@ public function addStart(int $now): API\SpanOptions
     public function toSpan(): API\Span
     {
         $span = $this->tracer->getActiveSpan();
-        $context = $span->getContext()->IsValidContext()
+        $context = $span && $span->getContext()->IsValidContext()
```

Comment:
Do we need to check for a span and then check to make sure it has a valid context?  Could `$span->getContext()->IsValidContext()` be sufficient here?

Reply:
yes, because I changed getActiveSpan and it may return null, now.

Reply:
Are other tracers using a noop span or their equivalent of null?

Reply:
No, but since we don't create a global span when Tracer is instantiated, then the getActiveSpan may return null. or should I throw an exception instead?

Reply:
IMO this should return `null` or a no-op span; no exception. We can make a no-op span on demand; no need to store it globally. I'm curious what other tracers do (in Open Tracing it would be a no-op span, FYI).